h2o-core/src/main/java/water/server/ServletUtils.java

```diff
@@ -172,6 +174,10 @@ public static boolean isXhrRequest(final HttpServletRequest request) {
     return "XMLHttpRequest".equals(requestedWithHeader);
   }
 
+  public static boolean isTraceRequest(final HttpServletRequest request) {
+    return TRACE_METHOD.equalsIgnoreCase(request.getMethod());
```

Comment:
Ignoring case is necessary. Some implementations actually might screw up the case. I've seen this many times.